Auto update failing for vulkan-validation-layers

Hi, I'm Termux 🤖.

I'm here to help you update your Termux packages.

I've tried to update the [vulkan-validation-layers](https://github.com/termux/termux-packages/blob/c5ec411ffe29b84e8e5fc013e366a23987e55ddd/packages/vulkan-validation-layers) package, but it failed.

Here's the output of the update script:
<details>
<summary>Show log</summary>
<pre lang="console">
INFO: Updating vulkan-validation-layers [Current version: 1.3.297]
INFO: package being updated to 1.3.302.
Downloading https://github.com/KhronosGroup/Vulkan-ValidationLayers/archive/refs/tags/v1.3.302.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 3232k    0 3232k    0     0  3837k      0 --:--:-- --:--:-- --:--:-- 3837k
100 5458k    0 5458k    0     0  4530k      0 --:--:--  0:00:01 --:--:-- 6151k

You are about to commit these changes:

--------------------
diff --git a/packages/vulkan-validation-layers/build.sh b/packages/vulkan-validation-layers/build.sh
index 0dd9058dd0..783d396319 100644
--- a/packages/vulkan-validation-layers/build.sh
+++ b/packages/vulkan-validation-layers/build.sh
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://github.com/KhronosGroup/Vulkan-ValidationLayers
 TERMUX_PKG_DESCRIPTION="Vulkan Validation Layers"
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="1.3.297"
+TERMUX_PKG_VERSION="1.3.302"
 TERMUX_PKG_SRCURL=https://github.com/KhronosGroup/Vulkan-ValidationLayers/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=11220ccc139f5517b3a3aa2e2ef0d680da314940c9bb98c7f3857a871bdb2205
+TERMUX_PKG_SHA256=78c49ebbe65c1034190d5d9ec57adea47833675d7f8b3b41fff86291a1e8fead
 TERMUX_PKG_DEPENDS="libc++, vulkan-loader"
 TERMUX_PKG_BUILD_DEPENDS="libwayland, libx11, libxcb, libxrandr, spirv-headers, spirv-tools, vulkan-headers (=${TERMUX_PKG_VERSION}), vulkan-utility-libraries (=${TERMUX_PKG_VERSION})"
 TERMUX_PKG_ANTI_BUILD_DEPENDS="vulkan-loader"
--------------------

bump(main/vulkan-validation-layers): 1.3.302


Not committing to Git!

INFO: Trying to build package.
INFO: Package vulkan-validation-layers exists in main repo.
Running container 'termux-package-builder' from image 'ghcr.io/termux/package-builder'...
termux - building vulkan-validation-layers for arch aarch64...
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13134  100 13134    0     0   155k      0 --:--:-- --:--:-- --:--:--  156k
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   3916      0 --:--:-- --:--:-- --:--:--  3929
gpg: Signature made Tue 26 Nov 2024 03:12:36 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13320  100 13320    0     0  64342      0 --:--:-- --:--:-- --:--:-- 64347
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   3904      0 --:--:-- --:--:-- --:--:--  3910
gpg: Signature made Thu 21 Nov 2024 12:56:15 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13120  100 13120    0     0  63517      0 --:--:-- --:--:-- --:--:-- 63689
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   3523      0 --:--:-- --:--:-- --:--:--  3529
gpg: Signature made Mon 25 Nov 2024 06:26:22 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading dependency libc++@27c if necessary...
Found libc++ in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libc++ to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libandroid-support@29 if necessary...
Found libandroid-support in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libandroid-support to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libexpat@2.6.4 if necessary...
Found libexpat in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libexpat to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libffi@3.4.6-1 if necessary...
Found libffi in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libffi to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libiconv@1.17 if necessary...
Found libiconv in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libiconv to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency iconv@1.17 if necessary...
Found iconv in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting iconv to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency liblzma@5.6.3 if necessary...
Found liblzma in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting liblzma to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xz-utils@5.6.3 if necessary...
Found xz-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xz-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses@6.5.20240831-2 if necessary...
Found ncurses in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-utils@6.5.20240831-2 if necessary...
Found ncurses-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-ui-libs@6.5.20240831-2 if necessary...
Found ncurses-ui-libs in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses-ui-libs to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-ui-libs-static@6.5.20240831-2 if necessary...
Found ncurses-ui-libs-static in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses-ui-libs-static to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency readline@8.2.13 if necessary...
Found readline in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting readline to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency gdbm@1.24 if necessary...
Found gdbm in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting gdbm to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libandroid-posix-semaphore@0.1-3 if necessary...
Found libandroid-posix-semaphore in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libandroid-posix-semaphore to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libbz2@1.0.8-6 if necessary...
Found libbz2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libbz2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency bzip2@1.0.8-6 if necessary...
Found bzip2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting bzip2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ca-certificates@1:2024.09.24 if necessary...
Found ca-certificates in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ca-certificates to /home/builder/.termux-build/_cache-all...
Downloading dependency ca-certificates-java@1:2024.09.24 if necessary...
Found ca-certificates-java in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ca-certificates-java to /home/builder/.termux-build/_cache-all...
Downloading dependency zlib@1.3.1 if necessary...
Found zlib in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting zlib to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency openssl@1:3.3.2 if necessary...
Found openssl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting openssl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency openssl-tool@1:3.3.2 if necessary...
Found openssl-tool in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting openssl-tool to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libcrypt@0.2-5 if necessary...
Found libcrypt in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libcrypt to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency tcl@8.6.14 if necessary...
Found tcl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting tcl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libsqlite@3.47.1 if necessary...
Found libsqlite in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libsqlite to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency sqlite@3.47.1 if necessary...
Found sqlite in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting sqlite to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libsqlite-tcl@3.47.1 if necessary...
Found libsqlite-tcl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libsqlite-tcl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency brotli@1.1.0 if necessary...
Found brotli in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting brotli to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libpng@1.6.44 if necessary...
Found libpng in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libpng to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency freetype@2.13.3 if necessary...
Found freetype in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting freetype to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ttf-dejavu@2.37-8 if necessary...
Found ttf-dejavu in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ttf-dejavu to /home/builder/.termux-build/_cache-all...
Downloading dependency fontconfig@2.15.0 if necessary...
Found fontconfig in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting fontconfig to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency fontconfig-utils@2.15.0 if necessary...
Found fontconfig-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting fontconfig-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xorg-util-macros@1.20.2 if necessary...
Found xorg-util-macros in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xorg-util-macros to /home/builder/.termux-build/_cache-all...
Downloading dependency xorgproto@2024.1 if necessary...
Found xorgproto in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xorgproto to /home/builder/.termux-build/_cache-all...
Downloading dependency libxau@1.0.11 if necessary...
Found libxau in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxau to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxdmcp@1.1.5 if necessary...
Found libxdmcp in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxdmcp to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xcb-proto@1.17.0-1 if necessary...
Found xcb-proto in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xcb-proto to /home/builder/.termux-build/_cache-all...
Downloading dependency python-xcbgen@1.17.0-1 if necessary...
Found python-xcbgen in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting python-xcbgen to /home/builder/.termux-build/_cache-all...
Downloading dependency libxcb@1.17.0 if necessary...
Found libxcb in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxcb to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xtrans@1.5.2 if necessary...
Found xtrans in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xtrans to /home/builder/.termux-build/_cache-all...
Downloading dependency libx11@1.8.10 if necessary...
Found libx11 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libx11 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxrender@0.9.11 if necessary...
Found libxrender in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxrender to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxft@2.3.8 if necessary...
Found libxft in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxft to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxext@1.3.6 if necessary...
Found libxext in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxext to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxss@1.2.4 if necessary...
Found libxss in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxss to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency tk@8.6.14 if necessary...
Found tk in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting tk to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency python@3.12.7-1 if necessary...
Found python in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting python to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency python-tkinter@3.12.7-1 if necessary...
Found python-tkinter in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting python-tkinter to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency python-ensurepip-wheels@3.12.7-1 if necessary...
Found python-ensurepip-wheels in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting python-ensurepip-wheels to /home/builder/.termux-build/_cache-all...
Downloading dependency libxml2@2.13.5 if necessary...
Found libxml2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxml2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxml2-utils@2.13.5 if necessary...
Found libxml2-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxml2-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxml2-python@2.13.5 if necessary...
Found libxml2-python in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxml2-python to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxml2-python-static@2.13.5 if necessary...
Found libxml2-python-static in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxml2-python-static to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libwayland@1.23.1-1 if necessary...
Found libwayland in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libwayland to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libwayland-cross-scanner@1.23.1-1 if necessary...
Found libwayland-cross-scanner in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libwayland-cross-scanner to /home/builder/.termux-build/_cache-all...
Downloading dependency libxrandr@1.5.4 if necessary...
Found libxrandr in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxrandr to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency spirv-headers@1.3.296.0 if necessary...
Found spirv-headers in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/s/spirv-headers/spirv-headers_1.3.296.0_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
 22  130k   22 30615    0     0  35829      0  0:00:03 --:--:--  0:00:03 35807
100  130k  100  130k    0     0   128k      0  0:00:01  0:00:01 --:--:--  128k
extracting spirv-headers to /home/builder/.termux-build/_cache-all...
Downloading dependency spirv-tools@1.3.296.0 if necessary...
Found spirv-tools in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/s/spirv-tools/spirv-tools_1.3.296.0_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0 1806k    0  1360    0     0   1575      0  0:19:34 --:--:--  0:19:34  1574
 92 1806k   92 1677k    0     0  1003k      0  0:00:01  0:00:01 --:--:-- 1002k
100 1806k  100 1806k    0     0  1079k      0  0:00:01  0:00:01 --:--:-- 1079k
extracting spirv-tools to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency vulkan-headers@1.3.302 if necessary...
Found vulkan-headers in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting vulkan-headers to /home/builder/.termux-build/_cache-all...
Downloading dependency vulkan-utility-libraries@1.3.302 if necessary...
Found vulkan-utility-libraries in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/v/vulkan-utility-libraries/vulkan-utility-libraries_1.3.302_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  6  496k    6 30685    0     0  54481      0  0:00:09 --:--:--  0:00:09 54406
100  496k  100  496k    0     0   462k      0  0:00:01  0:00:01 --:--:--  463k
extracting vulkan-utility-libraries to /home/builder/.termux-build/_cache-aarch64...
Downloading https://github.com/KhronosGroup/Vulkan-ValidationLayers/archive/refs/tags/v1.3.302.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 12827    0 12827    0     0  59328      0 --:--:-- --:--:-- --:--:-- 59328
100 5458k    0 5458k    0     0  5669k      0 --:--:-- --:--:-- --:--:-- 7300k
Applying patch: 0001-no-android.patch
Applying patch: 0002-getenv.patch
Applying patch: 0003-layers-utils-android_ndk_types.h.patch
-- Android: Targeting API '24' with architecture 'arm64', ABI 'arm64-v8a', and processor 'aarch64'
-- The CXX compiler identification is Clang 18.0.3
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /home/builder/.termux-build/_cache/android-r27c-api-24-v1/bin/clang++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Could NOT find robin_hood (missing: robin_hood_DIR)
-- Configuring done (0.4s)
-- Generating done (0.0s)
CMake Warning:
  Manually-specified variables were not used by the project:

    BUILD_TESTING
    CMAKE_C_FLAGS
    CMAKE_USE_SYSTEM_LIBRARIES
    DOXYGEN_EXECUTABLE


-- Build files have been written to: /home/builder/.termux-build/vulkan-validation-layers/build
[1/186] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/vulkan/generated/error_location_helper.cpp.o
[2/186] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/error_message/error_location.cpp.o
[3/186] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/vulkan/generated/pnext_chain_extraction.cpp.o
[4/186] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/vulkan/generated/feature_requirements_helper.cpp.o
[5/186] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/vulkan/generated/vk_object_types.cpp.o
[6/186] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/error_message/logging.cpp.o
[7/186] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/utils/convert_utils.cpp.o
[8/186] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/utils/hash_util.cpp.o
[9/186] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/utils/image_layout_utils.cpp.o
[10/186] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/utils/vk_layer_extension_utils.cpp.o
[11/186] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/vulkan/generated/vk_extension_helper.cpp.o
[12/186] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/utils/ray_tracing_utils.cpp.o
[13/186] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/utils/vk_struct_compare.cpp.o
[14/186] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/utils/vk_layer_utils.cpp.o
[15/186] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/vk_layer_config.cpp.o
FAILED: layers/CMakeFiles/VkLayer_utils.dir/vk_layer_config.cpp.o 
/home/builder/.termux-build/_cache/android-r27c-api-24-v1/bin/clang++ --target=aarch64-none-linux-android --gcc-toolchain=/home/builder/.termux-build/_cache/android-r27c-api-24-v1 --sysroot=/home/builder/.termux-build/_cache/android-r27c-api-24-v1/sysroot -DVK_ENABLE_BETA_EXTENSIONS -DVK_USE_PLATFORM_WAYLAND_KHR -DVK_USE_PLATFORM_XCB_KHR -DVK_USE_PLATFORM_XLIB_KHR -DVK_USE_PLATFORM_XLIB_XRANDR_EXT -I/home/builder/.termux-build/vulkan-validation-layers/src/layers/. -I/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan -isystem /home/builder/.termux-build/vulkan-validation-layers/src/layers/external -fstack-protector-strong -Oz --target=aarch64-linux-android24  -I/data/data/com.termux/files/usr/include -O3 -DNDEBUG -std=c++17 -fPIC -fvisibility=hidden -fvisibility-inlines-hidden -Wall -Wextra -Wpointer-arith -Wconversion -Wimplicit-fallthrough -Wstring-conversion -Wno-sign-conversion -Wno-implicit-int-conversion -MD -MT layers/CMakeFiles/VkLayer_utils.dir/vk_layer_config.cpp.o -MF layers/CMakeFiles/VkLayer_utils.dir/vk_layer_config.cpp.o.d -o layers/CMakeFiles/VkLayer_utils.dir/vk_layer_config.cpp.o -c /home/builder/.termux-build/vulkan-validation-layers/src/layers/vk_layer_config.cpp
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vk_layer_config.cpp:100:5: error: use of undeclared identifier 'assert'
  100 |     assert(false && "Not supported on android");
      |     ^
1 error generated.
[16/186] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/vulkan/generated/vk_dispatch_table_helper.cpp.o
[17/186] Building CXX object layers/gpu/spirv/CMakeFiles/gpu_av_spirv.dir/descriptor_indexing_oob_pass.cpp.o
FAILED: layers/gpu/spirv/CMakeFiles/gpu_av_spirv.dir/descriptor_indexing_oob_pass.cpp.o 
/home/builder/.termux-build/_cache/android-r27c-api-24-v1/bin/clang++ --target=aarch64-none-linux-android --gcc-toolchain=/home/builder/.termux-build/_cache/android-r27c-api-24-v1 --sysroot=/home/builder/.termux-build/_cache/android-r27c-api-24-v1/sysroot -DVK_ENABLE_BETA_EXTENSIONS -DVK_USE_PLATFORM_WAYLAND_KHR -DVK_USE_PLATFORM_XCB_KHR -DVK_USE_PLATFORM_XLIB_KHR -DVK_USE_PLATFORM_XLIB_XRANDR_EXT -I/home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/. -I/home/builder/.termux-build/vulkan-validation-layers/src/layers -I/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan -I/home/builder/.termux-build/vulkan-validation-layers/src/layers/. -fstack-protector-strong -Oz --target=aarch64-linux-android24  -I/data/data/com.termux/files/usr/include -O3 -DNDEBUG -std=c++17 -fPIC -fvisibility=hidden -fvisibility-inlines-hidden -Wall -Wextra -Wpointer-arith -Wconversion -Wimplicit-fallthrough -Wstring-conversion -Wno-sign-conversion -Wno-implicit-int-conversion -MD -MT layers/gpu/spirv/CMakeFiles/gpu_av_spirv.dir/descriptor_indexing_oob_pass.cpp.o -MF layers/gpu/spirv/CMakeFiles/gpu_av_spirv.dir/descriptor_indexing_oob_pass.cpp.o.d -o layers/gpu/spirv/CMakeFiles/gpu_av_spirv.dir/descriptor_indexing_oob_pass.cpp.o -c /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/descriptor_indexing_oob_pass.cpp
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/descriptor_indexing_oob_pass.cpp:18:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/module.h:22:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/type_manager.h:19:
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:314:19: error: no member named 'OpAllocateNodePayloadsAMDX' in namespace 'spv'; did you mean 'OpFinalizeNodePayloadsAMDX'?
  314 |         case spv::OpAllocateNodePayloadsAMDX:
      |              ~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~
      |                   OpFinalizeNodePayloadsAMDX
/data/data/com.termux/files/usr/include/spirv/unified1/spirv.hpp:1849:5: note: 'OpFinalizeNodePayloadsAMDX' declared here
 1849 |     OpFinalizeNodePayloadsAMDX = 5075,
      |     ^
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/descriptor_indexing_oob_pass.cpp:18:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/module.h:22:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/type_manager.h:19:
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:316:19: error: no member named 'OpNodePayloadArrayLengthAMDX' in namespace 'spv'
  316 |         case spv::OpNodePayloadArrayLengthAMDX:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:317:19: error: no member named 'OpIsNodePayloadValidAMDX' in namespace 'spv'
  317 |         case spv::OpIsNodePayloadValidAMDX:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:340:19: error: no member named 'OpCooperativeMatrixConvertNV' in namespace 'spv'
  340 |         case spv::OpCooperativeMatrixConvertNV:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:349:19: error: no member named 'OpCooperativeMatrixReduceNV' in namespace 'spv'
  349 |         case spv::OpCooperativeMatrixReduceNV:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:350:19: error: no member named 'OpCooperativeMatrixLoadTensorNV' in namespace 'spv'; did you mean 'OpCooperativeMatrixLoadNV'?
  350 |         case spv::OpCooperativeMatrixLoadTensorNV:
      |              ~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                   OpCooperativeMatrixLoadNV
/data/data/com.termux/files/usr/include/spirv/unified1/spirv.hpp:1906:5: note: 'OpCooperativeMatrixLoadNV' declared here
 1906 |     OpCooperativeMatrixLoadNV = 5359,
      |     ^
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/descriptor_indexing_oob_pass.cpp:18:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/module.h:22:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/type_manager.h:19:
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:351:19: error: no member named 'OpCooperativeMatrixPerElementOpNV' in namespace 'spv'; did you mean 'OpCooperativeMatrixLengthNV'?
  351 |         case spv::OpCooperativeMatrixPerElementOpNV:
      |              ~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                   OpCooperativeMatrixLengthNV
/data/data/com.termux/files/usr/include/spirv/unified1/spirv.hpp:1909:5: note: 'OpCooperativeMatrixLengthNV' declared here
 1909 |     OpCooperativeMatrixLengthNV = 5362,
      |     ^
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/descriptor_indexing_oob_pass.cpp:18:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/module.h:22:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/type_manager.h:19:
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:352:19: error: no member named 'OpCreateTensorLayoutNV' in namespace 'spv'
  352 |         case spv::OpCreateTensorLayoutNV:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:353:19: error: no member named 'OpTensorLayoutSetDimensionNV' in namespace 'spv'
  353 |         case spv::OpTensorLayoutSetDimensionNV:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:354:19: error: no member named 'OpTensorLayoutSetStrideNV' in namespace 'spv'
  354 |         case spv::OpTensorLayoutSetStrideNV:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:355:19: error: no member named 'OpTensorLayoutSliceNV' in namespace 'spv'
  355 |         case spv::OpTensorLayoutSliceNV:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:356:19: error: no member named 'OpTensorLayoutSetClampValueNV' in namespace 'spv'
  356 |         case spv::OpTensorLayoutSetClampValueNV:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:357:19: error: no member named 'OpCreateTensorViewNV' in namespace 'spv'
  357 |         case spv::OpCreateTensorViewNV:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:358:19: error: no member named 'OpTensorViewSetDimensionNV' in namespace 'spv'
  358 |         case spv::OpTensorViewSetDimensionNV:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:359:19: error: no member named 'OpTensorViewSetStrideNV' in namespace 'spv'
  359 |         case spv::OpTensorViewSetStrideNV:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:361:19: error: no member named 'OpTensorViewSetClipNV' in namespace 'spv'
  361 |         case spv::OpTensorViewSetClipNV:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:362:19: error: no member named 'OpTensorLayoutSetBlockSizeNV' in namespace 'spv'
  362 |         case spv::OpTensorLayoutSetBlockSizeNV:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:363:19: error: no member named 'OpCooperativeMatrixTransposeNV' in namespace 'spv'; did you mean 'OpCooperativeMatrixStoreNV'?
  363 |         case spv::OpCooperativeMatrixTransposeNV:
      |              ~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                   OpCooperativeMatrixStoreNV
/data/data/com.termux/files/usr/include/spirv/unified1/spirv.hpp:1907:5: note: 'OpCooperativeMatrixStoreNV' declared here
 1907 |     OpCooperativeMatrixStoreNV = 5360,
      |     ^
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/descriptor_indexing_oob_pass.cpp:18:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/module.h:22:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/type_manager.h:19:
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:408:19: error: no member named 'OpArithmeticFenceEXT' in namespace 'spv'
  408 |         case spv::OpArithmeticFenceEXT:
      |              ~~~~~^
fatal error: too many errors emitted, stopping now [-ferror-limit=]
20 errors generated.
[18/186] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/vulkan/generated/vk_function_pointers.cpp.o
ninja: build stopped: subcommand failed.
ERROR: failed to build.</pre>
</details>

<hr>
<i>
Above error occured when I last tried to update at 2024-11-26 06:20:09 UTC.<br>
Run ID: <a href="https://github.com/termux/termux-packages/actions/runs/12024835151">12024835151</a><br><br>
<b>Note:</b> Automatic updates will be disabled until this issue is resolved.<br>
</i>

--- a/packages/vulkan-validation-layers/0004-setenv.patch
+++ b/packages/vulkan-validation-layers/0004-setenv.patch
@@ -1,0 +1,11 @@
+--- a/layers/vk_layer_config.cpp
++++ b/layers/vk_layer_config.cpp
+@@ -90,7 +90,7 @@
+ }
+ 
+ void SetEnvironment(const char *variable, const char *value) {
+-#if !defined(__ANDROID__) && !defined(_WIN32)
++#if !defined(__ANDROID__) && !defined(_WIN32) || defined(__TERMUX__)
+     setenv(variable, value, 1);
+ #elif defined(_WIN32)
+     SetEnvironmentVariable(variable, value);

--- a/packages/vulkan-validation-layers/build.sh
+++ b/packages/vulkan-validation-layers/build.sh
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://github.com/KhronosGroup/Vulkan-ValidationLayers
 TERMUX_PKG_DESCRIPTION="Vulkan Validation Layers"
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="1.3.297"
+TERMUX_PKG_VERSION="1.4.304"
 TERMUX_PKG_SRCURL=https://github.com/KhronosGroup/Vulkan-ValidationLayers/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=11220ccc139f5517b3a3aa2e2ef0d680da314940c9bb98c7f3857a871bdb2205
+TERMUX_PKG_SHA256=fd889b0135b7770a3d9819ed1273009ddd654b9715b09c0af150eca8ffc15eb2
 TERMUX_PKG_DEPENDS="libc++, vulkan-loader"
 TERMUX_PKG_BUILD_DEPENDS="libwayland, libx11, libxcb, libxrandr, spirv-headers, spirv-tools, vulkan-headers (=${TERMUX_PKG_VERSION}), vulkan-utility-libraries (=${TERMUX_PKG_VERSION})"
 TERMUX_PKG_ANTI_BUILD_DEPENDS="vulkan-loader"

